src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -35,23 +36,24 @@ trait ClusterComponent extends LeoComponent {
     def googleId =              column[UUID]              ("googleId",              O.Unique)
     def googleProject =         column[String]            ("googleProject",         O.Length(254))
     def googleServiceAccount =  column[String]            ("googleServiceAccount",  O.Length(254))
-    def googleBucket =          column[String]            ("googleBucket",          O.Length(254))
+    def googleBucket =          column[String]            ("googleBucket",          O.Length(1024))
     def operationName =         column[String]            ("operationName",         O.Length(254))
     def status =                column[String]            ("status",                O.Length(254))
     def hostIp =                column[Option[String]]    ("hostIp",                O.Length(254))
     def createdDate =           column[Timestamp]         ("createdDate",           O.SqlType("TIMESTAMP(6)"))
     def destroyedDate =         column[Option[Timestamp]] ("destroyedDate",         O.SqlType("TIMESTAMP(6)"))
     def jupyterExtensionUri =   column[Option[String]]    ("jupyterExtensionUri",   O.Length(1024))
+    def initBucket =            column[String]            ("initBucket",            O.Length(1024))
 
     def uniqueKey = index("IDX_CLUSTER_UNIQUE", (googleProject, clusterName), unique = true)
 
-    def * = (id, clusterName, googleId, googleProject, googleServiceAccount, googleBucket, operationName, status, hostIp, createdDate, destroyedDate, jupyterExtensionUri) <> (ClusterRecord.tupled, ClusterRecord.unapply)
+    def * = (id, clusterName, googleId, googleProject, googleServiceAccount, googleBucket, operationName, status, hostIp, createdDate, destroyedDate, jupyterExtensionUri, initBucket) <> (ClusterRecord.tupled, ClusterRecord.unapply)
   }
 
   object clusterQuery extends TableQuery(new ClusterTable(_)) {
 
-    def save(cluster: Cluster): DBIO[Cluster] = {
-      (clusterQuery returning clusterQuery.map(_.id) += marshalCluster(cluster)) flatMap { clusterId =>
+    def save(cluster: Cluster, initBucket: String): DBIO[Cluster] = {
```

Comment:
Why not just store the initBucket inside the `Cluster` case class?

Reply:
Probs because we return Cluster to the user - and we don't want to return initBucket?

Reply:
Yup. The init bucket is full of secrets -- don't want to accidentally leak it.